src/components/common/Text/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { createElement, PropsWithChildren } from 'react';
+
+type Props = {
+  as?: string;
+  highlight?: boolean;
+  className?: string;
+  loading?: boolean;
+};
```

Comment:
모두 옵셔널이면 전체를 유틸 타입으로 감싸서 표현하는게 낫지 않을까요?

Reply:
요런식으로 `Partial`을 이야기하시는걸까요? 👀 혹여 추가되는 props들도 옵셔널일 확률이 높은 컴포넌트에서는 유용할 것 같아요!
```suggestion
type Props = Partial<{
  as: string;
  highlight: boolean;
  className: string;
  loading: boolean;
}>;
```